acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/privacy/PrivacyNet.java

```diff
@@ -228,7 +228,7 @@ public PrivacyNode makeNode(
                 name, generatePrivacyParameters(orion), keyFilePath, orion);
       } else {
         node =
-            pantheonNodeFactory.createIbftNodePrivacyEnabled(
+            pantheonNodeFactory.createIbftTwoNodePrivacyEnabled(
```

Comment:
Not for here - but that's a horrible interface, think it might be doing 2 things (i.e. IBFT and Privacy) - maybe it should have been a builder?

Reply:
💯  Yes